Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL = help
+
+REAL_ENV = dev
```

Comment:
if only we had a name for this environment categories (`dev`|`prod`), since they map to an higher number of unknown environments e.g. `continuumtest` in eLife, `staging` and `production` somewhere else, etc
For example, Android has various name like [`flavors` and `variants`](https://developer.android.com/studio/build/), this would probably be a `build type` in that terminology. Doesn't sound very specific but it's a wider discussion.